tests/MoneyFormatterTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Tests;
+
+use App\FormatterService\MoneyFormatter;
+use App\FormatterService\NumberFormatterInterface;
+use PHPUnit\Framework\TestCase;
+
+class MoneyFormatterTest extends TestCase
+{
+    public function providerNumber()
+    {
+        return [
+            ["$2.84M", "2.84M €", "2.84M", 2835779],
+            ["$211.99", "211.99 €", "211.99", 211.99],
+        ];
+    }
+
+    /**
+     * @dataProvider providerNumber
+     * @param $usd
```

Comment:
Parametrai be aprašymo ir be tipų duoda mažai naudos ir tiesiog reikia daugiau kodo perskaityti (ir po pakeitimų vėl pergeneruoti).

Todėl verta arba dėti parametrams tipus, arba detaliau aprašyti, ką gi tie parametrai iš tikrųjų duoda.
Pvz.:
```php
    public function testMoneyFormat(string $usd, string $eur, string $value, float $number)
```
arba
```php
 /**
     * @dataProvider providerNumber
     * @param string $usd Expected USD formatted value
     * @param string $eur Expected EUR formatted value
     * @param string $value Expected number value
     * @param float $number Input number
     */
```

Plačiau:
* https://symfonycasts.com/screencast/php7/scalar-type-hints
* http://docs.phpdoc.org/references/phpdoc/tags/param.html